framework/include/executioners/Executioner.h

```diff
@@ -113,6 +113,12 @@ class Executioner : public MooseObject,
   /// Augmented Picard convergence check that to be called by PicardSolve and can be overridden by derived executioners
   virtual bool augmentedPicardConvergenceCheck() const { return false; }
 
+  /**
+   * Get the verbose output flag
+   * @return The verbose output flag
+   */
+  const bool & verbose() const { return _verbose; }
```

Comment:
Returning a reference to a member that cannot change is just weird. Like why would you do this? It's not wrong, but why?

Reply:
Habit mostly. It makes sense to do when you don't want to copy some large object, but it's silly to do for a bool as I've done here.